helm/values.yaml

```diff
@@ -0,0 +1,97 @@
+# Default values for selfcare-uservice-party-management.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: selc
+
+replicaCount: 1
+
+image:
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+commandConfig:
+  command: [ "bin/pdnd-interop-uservice-party-management" ]
+  args: [ "-Dconfig.override_with_env_vars=true" ]
+
+imagePullSecrets:
+  - name: regcred
+
+nameOverride: ""
+fullnameOverride: "pdnd-interop-uservice-party-management"
+
+containers:
+  portNumber: 8088
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  runAsUser: 1001
+  allowPrivilegeEscalation: false
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+
+service:
+  type: ClusterIP
+  port: 8088
+  targetPort: http
+  monitoringPort: 9095
+  monitoringTargetPort: monitoring
+
+ingress:
+  enabled: true
+  className: ""
+  annotations:
+   kubernetes.io/ingress.class: "nginx"
+   nginx.ingress.kubernetes.io/rewrite-target: "/pdnd-interop-uservice-$1/0.1/$2"
+   nginx.ingress.kubernetes.io/ssl-redirect: "false"
+   nginx.ingress.kubernetes.io/use-regex: "true"
+   nginx.ingress.kubernetes.io/enable-cors: "true"
+   nginx.ingress.kubernetes.io/cors-allow-headers: "local.cors.headers"
+   nginx.ingress.kubernetes.io/cors-allow-origin: "local.cors.origins"
+  hosts:
+    - host: ""
+      paths:
+        - path: "/(party-management)/v1/(.*)"
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
```

Comment:
Please enabled this attribute to be able to manage the load, and avoid problems with the pod
https://github.com/pagopa/template-microservizio-k8s/blob/e7a198f1125ac53b7fc61154851de26dac4890fa/helm/values-prod.yaml#L26

Reply:
in order to provide a limit on the resources we should talk to every responsible of BE application to ask for suggestions